packages/khalti_flutter/lib/src/widget/khalti_progress_indicator.dart

```diff
@@ -16,12 +16,13 @@ class KhaltiProgressIndicator extends StatefulWidget {
   final double size;
 
   @override
-  _KhaltiProgressIndicatorState createState() {
-    return _KhaltiProgressIndicatorState();
+  KhaltiProgressIndicatorState createState() {
```

Comment:
Keep the state private. You can update the return type to `State<KhaltiProgressIndicator>` to get rid of the lint issue.